docs/topics/gradle/gradle-configure-project.md

```diff
@@ -386,6 +386,46 @@ integrationTestCompilation {
 Here, the `integrationTest` compilation is associated with the `main` compilation that gives access to `internal`
 objects from functional tests.
 
+### Configure with Java JPMS enabled
```

Comment:
Oracle doesn't like this abbreviation, maybe use "Java Modules" term?

Reply:
Even if they don't, they use it. Below we give a link to their blog: https://www.oracle.com/corporate/features/understanding-java-9-modules.html
Quotation: "In this article, I introduce the Java 9 Platform Module System (JPMS)".

Actually, I would replace it with "Java Modules (JPMS)", so it would be even more findable by users.